src/chord.js

```diff
@@ -79,32 +79,28 @@ const modeMap = {
 };
 
 // Alternate names for chords
-modeMap.Maj = modeMap.maj;
```

Comment:
Would this continue to let us use something like `CMaj` (I hope we are not limiting to using lower case only)